src/utils/AuthProvider.tsx

```diff
@@ -101,6 +102,15 @@ export default function AuthProvider({
         options,
       }); // will trigger onAuthStateChange to update the session
 
+      // This code is largely taken from https://github.com/orgs/supabase/discussions/1282#discussioncomment-5230475
+
+      // User exists, but is fake. See https://supabase.com/docs/reference/javascript/auth-signup
```

Comment:
nit: dont think these comments are true anymore, we should be able to get rid of them